Return False if network is not in networks

kinghuang opened the pull request:
If network is not in the list networks, return False for
is_external_network(). Otherwise, a KeyError occurs.

Resolves #6.

kinghuang committed:
Return False if network is not in networks

If network is not in the list networks, return False for
is_external_network(). Otherwise, a KeyError occurs.

Resolves #6.

ddrozdov commented:
I think we should better throw some error since it's a misconfiguration when a service uses a network that is not defined under the top-level `networks` key.

kinghuang commented:
Hmm, yes, that is probably better as an error. I'll change it to that.

kinghuang committed:
Change to an error if network is not defined

Instead of returning False, print a message to stderr if the named
network is not defined, and exit.
